app/src/main/java/org/asteroidos/sync/ble/MediaService.java

```diff
@@ -190,29 +229,24 @@ public void onQueueTitleChanged(CharSequence title) {
     @Override
     public void onActiveSessionsChanged(List<MediaController> controllers) {
         if (controllers.size() > 0) {
-            if (mMediaController != null) {
-                if (!controllers.get(0).getSessionToken().equals(mMediaController.getSessionToken())) {
-                    // Detach current controller
-                    mMediaController.unregisterCallback(mMediaCallback);
-                    Log.d("MediaService", "MediaController removed");
-                    mMediaController = null;
-
-                    // Attach new controller
-                    mMediaController = controllers.get(0);
-                    mMediaController.registerCallback(mMediaCallback);
-                    mMediaCallback.onMetadataChanged(mMediaController.getMetadata());
-                    if(mMediaController.getPlaybackState() != null)
-                        mMediaCallback.onPlaybackStateChanged(mMediaController.getPlaybackState());
-                    Log.d("MediaService", "MediaController set: " + mMediaController.getPackageName());
-                }
-            } else {
+            if (mMediaController != null && !controllers.get(0).getSessionToken().equals(mMediaController.getSessionToken())) {
+                // Detach current controller
+                mMediaController.unregisterCallback(mMediaCallback);
+                Log.d("MediaService", "MediaController removed");
+                mMediaController = null;
+            }
+
+            if(mMediaController == null) {
                 // Attach new controller
                 mMediaController = controllers.get(0);
                 mMediaController.registerCallback(mMediaCallback);
                 mMediaCallback.onMetadataChanged(mMediaController.getMetadata());
-                if(mMediaController.getPlaybackState() != null)
+                if (mMediaController.getPlaybackState() != null)
                     mMediaCallback.onPlaybackStateChanged(mMediaController.getPlaybackState());
-                Log.d("MediaService", "MediaController set: " + mMediaController.getPackageName());
+                Log.e("MediaService", "MediaController set: " + mMediaController.getPackageName());
```

Comment:
Any reason why you switched the log level to error?

Reply:
Oups. I made it Error so that it was standing out of my logs during debugging. Forgot to change it back. Will move it back to debug